src/components/PostJobPage/PostJobLayout.styles.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+	border: 1px solid black;
+	border-radius: 20px;
+	margin: 20px;
+	padding: 20px;
+`;
+
+export const TitleStyled = styled.p`
+	font-weight: 700;
+`;
+
+export const DescriptionDataStyled = styled.p`
+	height: 40px;
+	overflow: hidden;
+`;
+
+export const DateStyled = styled.div`
+	width: 200px;
+	height: 35px;
+	display: flex;
+	align-items: center;
+	background-color: gray;
+	color: white;
```

Comment:
same, all colors

Reply:
add constants in theme by styled components